src/logs.py

```diff
@@ -1,28 +1,42 @@
+# This code imports the `glob`, `os`, and `time` modules
```

Comment:
Please do not use descriptive comments for lines like this, it's obvious what is done here.